pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -653,6 +653,10 @@ func (c *Reconciler) makeConditionCheckContainer(rprt *resources.ResolvedPipelin
 	labels := getTaskrunLabels(pr, rprt.PipelineTask.Name)
 	labels[pipeline.GroupName+pipeline.ConditionCheckKey] = rcc.ConditionCheckName
 
+	for key, value := range rcc.Condition.ObjectMeta.Labels {
```

Comment:
Neat neat! Could we add a little blurb that the labels get added in https://github.com/tektoncd/pipeline/blob/master/docs/conditions.md ? :pray:

Reply:
I've added some cursory documentation.  I'm slightly concerned that this is exposing implementation details by referencing the TaskRun created for the Condition, but in this case it's probably acceptable.  We should also consider updating the main Readme with a reference to the Condition documentation.